src/CsharpClient/QuixStreams.Streaming/KafkaStreamingClient.cs

```diff
@@ -11,10 +11,46 @@
 
 namespace QuixStreams.Streaming
 {
+    public interface IKafkaStreamingClient
+    {
+        /// <summary>
+        /// Gets a topic consumer capable of subscribing to receive incoming streams.
+        /// </summary>
+        /// <param name="topic">Name of the topic.</param>
+        /// <param name="consumerGroup">The consumer group id to use for consuming messages. If null, consumer group is not used and only consuming new messages.</param>
+        /// <param name="options">The settings to use for committing</param>
+        /// <param name="autoOffset">The offset to use when there is no saved offset for the consumer group.</param>
+        /// <returns>Instance of <see cref="ITopicConsumer"/></returns>
+        ITopicConsumer GetTopicConsumer(string topic, string consumerGroup = null, CommitOptions options = null, AutoOffsetReset autoOffset = AutoOffsetReset.Latest);
+
+        /// <summary>
+        /// Gets a topic consumer capable of subscribing to receive non-quixstreams incoming messages. 
+        /// </summary>
+        /// <param name="topic">Name of the topic.</param>
+        /// <param name="consumerGroup">The consumer group id to use for consuming messages. If null, consumer group is not used and only consuming new messages.</param>
+        /// <param name="autoOffset">The offset to use when there is no saved offset for the consumer group.</param>
+        /// <returns>Instance of <see cref="ITopicConsumer"/></returns>
+        IRawTopicConsumer GetRawTopicConsumer(string topic, string consumerGroup = null, AutoOffsetReset? autoOffset = null);
+
+        /// <summary>
+        /// Gets a topic producer capable of publishing non-quixstreams messages.  
+        /// </summary>
+        /// <param name="topic">Name of the topic.</param>
+        /// <returns>Instance of <see cref="ITopicConsumer"/></returns>
```

Comment:
should be  `<see cref="IRawTopicProducer"...`
words to change `Raw` and `Producer`